app/sidekiq/hca/ezr_submission_job.rb

```diff
@@ -7,16 +7,29 @@ module HCA
   class EzrSubmissionJob
     include Sidekiq::Job
     include SentryLogging
+    include Common::Client::Concerns::Monitoring
     VALIDATION_ERROR = HCA::SOAPParser::ValidationError
 
     sidekiq_options retry: 14
 
     sidekiq_retries_exhausted do |msg, _e|
-      log_submission_failure(decrypt_form(msg['args'][0]))
-    end
-
-    def self.log_submission_failure(parsed_form)
-      Form1010Ezr::Service.new(nil).log_submission_failure(parsed_form)
+      parsed_form = decrypt_form(msg['args'][0])
+
+      StatsD.increment("#{Form1010Ezr::Service::STATSD_KEY_PREFIX}.failed_wont_retry")
+
+      if parsed_form.present?
+        PersonalInformationLog.create!(
+          data: parsed_form,
+          error_class: 'Form1010Ezr FailedWontRetry'
+        )
+
+        new.log_message_to_sentry(
```

Comment:
I'm not sure why, but without add the `new` to instantiate the class, my test was failing and I was getting:
```
  1) HCA::EzrSubmissionJob when retries are exhausted when the parsed form is present increments StatsD, creates a 'PersonalInformationLog' record, and logs a failure message to sentry
     Failure/Error: 
       log_message_to_sentry(
         '1010EZR total failure',
         :error,
         Form1010Ezr::Service.new(nil).veteran_initials(parsed_form),
         ezr: :total_failure
       )

     NoMethodError:
       undefined method `log_message_to_sentry' for class HCA::EzrSubmissionJob
```
If anyone knows of a better way to do this, please let me know.

Reply:
It turns out we needed to add the `SentryLogging` classes to the class itself, as opposed to instances of the class. We changed `include` to `extend` to achieve this.